<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>461</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>301</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Authentication</string>
      </property>
      <widget class="QLabel" name="l_username">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>69</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLabel" name="l_password">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>100</y>
         <width>71</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_username">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>144</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="le_password">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>146</width>
         <height>27</height>
        </rect>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_login">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>130</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPlainTextEdit" name="te_topics">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>431</width>
        <height>78</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_first">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_prev">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_next">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>140</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_last">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>140</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&gt;&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_goback">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Topics</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_done">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>190</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Done</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>221</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="qtpresentationresources.qrc">:/images/images/Qt_logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>221</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Qt Presentation for SE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>35</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>- Murtaza Raja</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>491</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>This application was compiled using Qt5.4 on Ubuntu Trusty Tahr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>491</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>source code available at github.com/murtraja/qt-presentation-for-se</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtpresentationresources.qrc"/>
 </resources>
 <connections/>
</ui>
